### DISCUSSION TITLE
Command Seperation

### DISCUSSION THREAD

**Discussion by thecodemonster-AI** [winglessraven/AMP-Discord-Bot#80] [Ideas]
I was thinking of having The start, stop, restart, and manage commands seperate so i can set permisions for each commands diffrent. meaning I can allow people to start the server without allowing them to manage it.

**Comment by winglessraven** [comment]
Enable this option to remove the bot name from the command - this will let you control permissions within Discord for each command... 
![image](https://github.com/winglessraven/AMP-Discord-Bot/assets/4540397/9f289c8c-7616-46c7-9acb-97c8085533a9)

**Reply by thecodemonster-AI** [reply]
it still show the commands as /ampserver not allowing me to set perms for commands even with remove bot name enabled

![image](https://github.com/winglessraven/AMP-Discord-Bot/assets/72108651/6f935367-9057-4218-913a-f5b2f2c789dd)

**Comment by winglessraven** [comment]
Toggle the bot active off and on again, then you need to wait for Discord to update the commands (can take upto 20mins)
